libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/NavigationView.kt

```diff
@@ -26,38 +26,56 @@ import com.mapbox.maps.plugin.gestures.OnMapLongClickListener
+import com.mapbox.navigation.dropin.component.recenter.RecenterViewModel
+import com.mapbox.navigation.dropin.component.routearrow.RouteArrowViewModel
+import com.mapbox.navigation.dropin.component.routearrow.RouteArrowViewModelFactory
+import com.mapbox.navigation.dropin.component.routeline.RouteLineViewModel
+import com.mapbox.navigation.dropin.component.routeline.RouteLineViewModelFactory
+import com.mapbox.navigation.dropin.component.routeoverview.RouteOverviewUIComponent
+import com.mapbox.navigation.dropin.component.routeoverview.RouteOverviewViewModel
+import com.mapbox.navigation.dropin.component.sound.SoundButtonViewModel
+import com.mapbox.navigation.dropin.component.sound.SoundUIComponent
+import com.mapbox.navigation.dropin.component.speedlimit.SpeedLimitUIComponent
+import com.mapbox.navigation.dropin.component.speedlimit.SpeedLimitViewModel
+import com.mapbox.navigation.dropin.component.tripprogress.TripProgressUIComponent
+import com.mapbox.navigation.dropin.component.tripprogress.TripProgressViewModel
 import com.mapbox.navigation.dropin.databinding.MapboxLayoutDropInViewBinding
-import com.mapbox.navigation.dropin.state.ContainerVisibilityState
 import com.mapbox.navigation.dropin.util.MapboxDropInUtils
-import com.mapbox.navigation.dropin.viewmodel.CameraViewModel
-import com.mapbox.navigation.dropin.viewmodel.ContainerVisibilityViewModel
-import com.mapbox.navigation.dropin.viewmodel.MapboxNavigationViewModel
-import com.mapbox.navigation.dropin.viewmodel.NavigationStateViewModel
-import com.mapbox.navigation.dropin.viewmodel.RouteArrowViewModel
-import com.mapbox.navigation.dropin.viewmodel.RouteLineViewModel
 import com.mapbox.navigation.ui.maneuver.view.MapboxManeuverView
+import com.mapbox.navigation.ui.maps.camera.state.NavigationCameraStateChangedObserver
 import com.mapbox.navigation.ui.maps.camera.view.MapboxRecenterButton
 import com.mapbox.navigation.ui.maps.camera.view.MapboxRouteOverviewButton
 import com.mapbox.navigation.ui.maps.location.NavigationLocationProvider
+import com.mapbox.navigation.ui.speedlimit.model.SpeedLimitFormatter
 import com.mapbox.navigation.ui.speedlimit.view.MapboxSpeedLimitView
+import com.mapbox.navigation.ui.tripprogress.model.DistanceRemainingFormatter
+import com.mapbox.navigation.ui.tripprogress.model.EstimatedTimeToArrivalFormatter
+import com.mapbox.navigation.ui.tripprogress.model.TimeRemainingFormatter
+import com.mapbox.navigation.ui.tripprogress.model.TripProgressUpdateFormatter
 import com.mapbox.navigation.ui.tripprogress.view.MapboxTripProgressView
 import com.mapbox.navigation.ui.voice.view.MapboxSoundButton
 import com.mapbox.navigation.utils.internal.ifNonNull
 import kotlinx.coroutines.ExperimentalCoroutinesApi
 import kotlinx.coroutines.flow.collect
 import kotlinx.coroutines.flow.flowOf
 import kotlinx.coroutines.launch
-import java.lang.IllegalStateException
 import java.lang.ref.WeakReference
 import java.util.concurrent.CopyOnWriteArraySet
 
```

Comment:
The `@ExperimentalCoroutinesApi` should be changed to `@OptIn(ExperimentalCoroutinesApi::class)`.